### PR TITLE
Br. Portuguese Translation - Update

### PR DESCRIPTION
**Pull request by GuilhermeDiasJ**
- A lot of grammatical errors were fixed;
- Several sentences were reformulated for a better understanding and a better quality of the translation.

### CODE REVIEW

**Comment by pgert** [comment]
You should include the mo-file in "bin > Langs >pt_BR" if you want this to have effect.

File: locales/pt_BR/pcsx2_Main.po [hunk 1]
```diff
@@ -1282,7 +1282,7 @@ msgid ""
 "Note: Duplicate/Rename/Create/Delete will NOT be reverted with 'Cancel'."
 msgstr ""
 "\n"
-"Nota: Duplicar/Renomear/Criar/Excluir NÃO será revertido com \"Cancelar\"."
+"Nota: duplicar/renomear/criar/eliminar NÃO será revertido com "Cancelar"."
```
Comment (pgert) [inline locales/pt_BR/pcsx2_Main.po]: Isn't "Duplicar/Renomear/Criar/Eliminar" better here, you think?

Comment (GuilhermeDiasJ) [reply locales/pt_BR/pcsx2_Main.po]: I left in lowercase because the grammar says after a colon, the lowercase letter should be used. However, if you think it's best to leave as it is in English, that's okay, as this is not a big mistake.

Comment (pgert) [reply locales/pt_BR/pcsx2_Main.po]: "Duplicar/Renomear/Criar/Eliminar" referes to the buttons, but are (here) also names of operations, and thus it's grammatically correct.

Comment (GuilhermeDiasJ) [reply locales/pt_BR/pcsx2_Main.po]: Ok, let's use "Duplicar/Renomear/Criar/Eliminar then.

**Comment by pgert** [comment]
Well?

Do you intend to update the po-file with "Duplicar/Renomear/Criar/Eliminar",
 and do you intend to include the mo-file in "bin > Langs > pt_BR"?

**Comment by GuilhermeDiasJ** [comment]
> Well?
> 
> Do you intend to update the po-file with "Duplicar/Renomear/Criar/Eliminar",
> and do you intend to include the mo-file in "bin > Langs > pt_BR"?

Yes, the file is now updated (Update 3).
However, how can I update the mo-file?

**Comment by pgert** [comment]
When you save a file in POedit, it also creates an mo-file.
Use that to replace the existing one in "bin > Langs > pt_BR" in your GitHub repository, the same repository you have been using for the po-file.
Then commit.

**Comment by GuilhermeDiasJ** [comment]
> When you save a file in POedit, it also creates an mo-file.
> Use that to replace the existing one in "bin > Langs > pt_BR" in your GitHub repository, the same repository you have been using for the po-file.
> Then commit.

Done, I guess? Sorry, I'm not familiar with GitHub.

**Comment by pgert** [comment]
Thats strange. The file is in you repo, but does not show here.
@lightningterror ?

File: locales/pt_BR/pcsx2_Main.po [hunk 2]
```diff
@@ -1274,15 +1274,15 @@ msgstr "Gerenciador de cartão de memória"
...
+msgstr "Arraste os cartões de ou para as portas do PS2"
 
 #: pcsx2/gui/Dialogs/McdConfigDialog.cpp:117
 msgid ""
 "\n"
 "Note: Duplicate/Rename/Create/Delete will NOT be reverted with 'Cancel'."
 msgstr ""
 "\n"
-"Nota: Duplicar/Renomear/Criar/Excluir NÃO será revertido com \"Cancelar\"."
+"Nota: Duplicar/Renomear/Criar/Eliminar NÃO será revertido com "Cancelar"."
```
Comment (tadanokojin) [inline locales/pt_BR/pcsx2_Main.po]: Why did you remove the escapes?

```
/home/travis/build/PCSX2/pcsx2/locales/pt_BR/pcsx2_Main.po:1285: keyword "Cancelar" unknown
/home/travis/build/PCSX2/pcsx2/locales/pt_BR/pcsx2_Main.po:1285:73: syntax error
/home/travis/build/PCSX2/pcsx2/locales/pt_BR/pcsx2_Main.po:2346: keyword "O" unknown
/home/travis/build/PCSX2/pcsx2/locales/pt_BR/pcsx2_Main.po:2346:38: syntax error
/home/travis/build/PCSX2/pcsx2/locales/pt_BR/pcsx2_Main.po:2457:31: syntax error
/home/travis/build/PCSX2/pcsx2/locales/pt_BR/pcsx2_Main.po:2457: keyword "s" unknown
/usr/bin/msgfmt: found 6 fatal errors
make[2]: *** [pt_BR__pcsx2_Main.gmo] Error 1
make[1]: *** [locales/CMakeFiles/translations_pcsx2_Main.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
```

**Comment by lightningterror** [comment]
@pgert He updated master branch instead of current one that is tied with the pr.

**Comment by pgert** [comment]
@GuilhermeDiasJ:
You know what, forget about the mo-file.
Once this PR is merged, I can make another one for said file.

File: locales/pt_BR/pcsx2_Main.po (hunk 2 above)
Comment (GuilhermeDiasJ) [reply locales/pt_BR/pcsx2_Main.po]: Done.

File: locales/pt_BR/pcsx2_Main.po [hunk 3]
```diff
@@ -2323,31 +2323,31 @@ msgstr ""
...
+"de rastreamento a qualquer momento com F10."
 
 #: pcsx2/gui/Panels/LogOptionsPanels.cpp:252
 msgid ""
 "Warning: Enabling trace logs is typically very slow, and is a leading cause of "
 "'What happened to my FPS?' problems. :)"
 msgstr ""
-"Aviso: Habilitar registros de rastreamento é normalmente muito lento e é uma das "
-"principais causas de problemas \"O que aconteceu com meu FPS?\". :)"
+"Aviso: a ativação dos registros de rastreamento é normalmente muito lenta e é a "
```
Comment (lightningterror) [inline locales/pt_BR/pcsx2_Main.po]: You forgot this one.

File: locales/pt_BR/pcsx2_Main.po [hunk 4]
```diff
@@ -2454,11 +2454,11 @@ msgstr "Falha ao copiar!"
 #: pcsx2/gui/Panels/MemoryCardListPanel.cpp:777
 #, c-format
 msgid "Memory card '%s' duplicated to '%s'."
-msgstr "O cartão de memória '%s' foi duplicado para '%s'."
+msgstr "O cartão de memória "%s" foi duplicado para "%s"."
```
Comment (lightningterror) [inline locales/pt_BR/pcsx2_Main.po]: Ditto.

Comment (GuilhermeDiasJ) [reply locales/pt_BR/pcsx2_Main.po]: Done.

**Comment by lightningterror** [comment]
@pgert you can take care of the .mo files now.
